pkg/openfeature/client.go

```diff
@@ -304,6 +304,10 @@ func mergeContexts(evaluationContexts ...EvaluationContext) EvaluationContext {
 	mergedCtx := evaluationContexts[0]
 
 	for i := 1; i < len(evaluationContexts); i++ {
+		if mergedCtx.TargetingKey == "" && evaluationContexts[i].TargetingKey != "" {
+			mergedCtx.TargetingKey = evaluationContexts[i].TargetingKey
```

Comment:
minor, but could add a break in here

Reply:
we want to keep merging the attributes, a break would end that

Reply:
ah!